src/libs/notifications.js

```diff
@@ -64,6 +64,15 @@ const NotificationDisplay = Utils.connectStore(notificationStore, 'notificationS
       ['error', () => 'error-standard']
     )
 
+    const ariaLabel = Utils.switchCase(type,
```

Comment:
I wonder if this could be included in the switch case at line 53 when the baseColor is being decided? Since they rely on the same type to make the switches it could help with some code de-duplication

Reply:
```suggestion
    const { baseColor, ariaLabel} = Utils.switchCase(type,
      ['success', () => ({baseColor: colors.success, ariaLabel:'success notification'})],
      ['info', () => ({baseColor: colors.accent, ariaLabel:'info notification'})],
      ['welcome', () => ({baseColor: colors.accent, ariaLabel:'welcome notification'})],
      ['warn', () => ({baseColor: colors.warning, ariaLabel:'warning notification'})],
      ['error', () =>  ({baseColor: colors.danger, ariaLabel:'error notification'})],
      [Utils.DEFAULT, () => ({baseColor: colors.accent, ariaLabel:'notification'})],
    )
```

Reply:
(this will need some spaces added to make lint happy)